HybridManagement/HybridManagement.psm1

```diff
@@ -5,6 +5,25 @@ using namespace System.Collections.Specialized
 using namespace System.Text
 using namespace System.Security
 
+# This module contains many cmdlets which may be used in different scenarios. Since the purpose 
+# of this module is to provide cmdlets that cross the cloud/on-premises boundary, you may want 
+# to take a look at what that cmdlets are doing prior to running them. For the ease of your 
+# inspection, we have grouped them into several regions:

```

Comment:
Should we use #region formatting for these sections?

#region General cmdlets
...
#endregion General cmdlets

etc.

Reply:
Cool, I thought these were C# only - didn't realize PowerShell had them too. Agree that we should do it this way, will change and then merge.